src/components/partnership/partnership.service.ts

```diff
@@ -449,6 +521,21 @@ export class PartnershipService {
       throw new NotFoundException('Could not find partnership');
     }
 
+    let mouStart = null;
+    let mouEnd = null;
+
+    // if user has access to project mou and there is no partnership override
+    if (result.projectMouStart && result.mouStartOveride !== null) {
+      mouStart = result.mouStartOverride ?? result.projectMouStart;
+    } else {
+      this.logger.info(`user doesn't have access or there is no override`);
```

Comment:
access to what? Also probably more of a debug message

Reply:
fixed.